mne_bids/read.py

```diff
@@ -191,6 +192,12 @@ def read_raw_bids(bids_fname, bids_root, verbose=True):
         Full name of the data file
     bids_root : str
         Path to root of the BIDS folder
+    allow_maxshield : bool | str (default False)
```

Comment:
read_raw_bids can read any input format not just fif. I am concerned about having a param that is only relevant for one file type. I would add an `extra_params` parameter that can be None or dict to be passed as **kwargs in read_raw_xxx.